modules/addons/variables.tf

```diff
@@ -0,0 +1,83 @@
+variable "create_vpc_cni_addon" {
+  type        = bool
+  description = "Controls if vpc cni addon should be deployed"
+  default     = true
+}
+
+variable "create_kube_proxy_addon" {
+  type        = bool
+  description = "Controls if kube proxy addon should be deployed"
+  default     = true
+}
+
+variable "create_coredns_addon" {
+  type        = bool
+  description = "Controls if coredns addon should be deployed"
+  default     = true
+}
+
+variable "cluster_name" {
+  type        = string
+  description = "Name of parent cluster"
+}
+
+
+variable "cluster_version" {
+  type        = string
+  description = "Kubernetes version to use for the EKS cluster."
+}
+
+variable "coredns_versions" {
+  # Versions are taken from https://docs.aws.amazon.com/eks/latest/userguide/managing-coredns.html#updating-coredns-add-on
+  type        = map(any)
+  description = "The CoreDns plugin version for the corresponding version"
+  default = {
+    "1.18" = "v1.8.3-eksbuild.1"
+    "1.19" = "v1.8.3-eksbuild.1"
+    "1.20" = "v1.8.3-eksbuild.1"
+  }
+}
+
+
+variable "kube_proxy_versions" {
+  # Versions are taken from https://docs.aws.amazon.com/eks/latest/userguide/managing-kube-proxy.html#updating-kube-proxy-add-on
+  type        = map(any)
```

Comment:
I think better will be `map(string)` just to not loose type over time

Reply:
Changed to `map(string)`